.github/workflows/tests.yml

```diff
@@ -79,25 +79,57 @@ jobs:
       # environment variables used by gh CLI
       # https://cli.github.com/manual/gh_help_environment
       GH_REPO: ${{ matrix.repository }}
-      GH_TOKEN: ${{ secrets.FLOWZONE_TOKEN }}
       GH_PROMPT_DISABLED: "true"
       GH_DEBUG: "true"
       GH_PAGER: "cat"
+      APP_ID: "${{ vars.APP_ID || 291899 }}"
+      INSTALLATION_ID: "${{ vars.INSTALLATION_ID || 34040165 }}"
+      TOKEN_SCOPE: >-
+        {
+          "administration": "write"
+        }
       pr_branch: "flowzone/${{ github.head_ref || github.ref_name }}"
       pr_title: "Test Flowzone PR ${{ github.event.pull_request.number }}"
 
     steps:
+      - name: Check secrets
```

Comment:
unfortunately we can't output the token from previous job, since it's masked, so have to duplicate code here.